### DISCUSSION TITLE
Spectator title should respect skin elements

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#25609] [Q&A]
Currently the title overlaps any skin element at the back of it.

![image](https://github.com/ppy/osu/assets/86934170/7975c170-a6a1-4785-b7f1-a15a8ce255d1)

**Comment by huyenden** [comment]
In my opinion, it is better to change the position of the skin element. If the player's title is in the back, that doesn't solve the problem if another player asks the same question as you in multiplayer.

**Reply by Loreos7** [reply]
I meant moving the title away from any skin element completely. Somewhere there are no skin elements at all. Like below, left or right.

**Reply by huyenden** [reply]
If you spectate 1 person, you can probably skip it. If you spectate in multiplayer without that title, you won't know who is playing.

Anyway, try waiting for ppy to reply

**Comment by Rekunan** [comment]
I think a better solution is to make that also a skinnable element.
